01week/rockPaperScissors.js

```diff
@@ -8,10 +8,30 @@ const rl = readline.createInterface({
 });
 
 
-function rockPaperScissors(hand1, hand2) {
+rockPaperScissors=(hand1, hand2)=>{
+  if (hand1 === hand2) {
+    return "It's a tie!";
```

Comment:
Also, convert your strings to uniform capitalization, remove whitespace and check for valid entries before evaluating. We developers can never assume users will put in the correct inputs.

Reply:
Just realized that what I submitted wasn't the finished project I thought it was... I can't find the branch that has it correct now. I will add it back in and do another pull request.